apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -111,26 +164,91 @@ class PassengerForm extends React.Component<Props, State> {
     this.setState({ id });
   };
 
+  validateYear = (year: number): boolean => {
+    return year > getYear(new Date()) - 110 && year < getYear(new Date()) - 18;
+  };
+
+  handleInvalidDate = (timePeriod, message) => {
+    this.setState(state => {
+      return {
+        ...state,
+        inputErrors: {
+          ...state.inputErrors,
+          [timePeriod]: message,
+        },
+      };
+    });
+  };
+
+  resetError = timePeriod => {
+    this.setState(state => {
+      return {
+        ...state,
+        inputErrors: {
+          ...state.inputErrors,
+          [timePeriod]: '',
+        },
+      };
+    });
+  };
+
+  handleDateValidation = () => {
+    const day = parseInt(this.state.date.day, 10);
+    const month = parseInt(this.state.date.month, 10);
+    const year = parseInt(this.state.date.year, 10);
+
+    let date = new Date();
+
+    if (!day) {
+      this.handleInvalidDate('day', 'Please fill in the field');
+    } else {
+      this.resetError('day');
+    }
+    if (!year) {
+      this.handleInvalidDate('year', 'Please fill in the field');
+      return false;
+    }
+    this.resetError('year');
+
+    const isYearValid = this.validateYear(year);
+    if (!isYearValid) {
```

Comment:
There should be different error message: 

![Screenshot 2019-06-06 at 11 24 42](https://user-images.githubusercontent.com/858321/59022275-bdd19880-884d-11e9-979a-55acf279f8d8.png)

Reply:
Yes, I should change validation here and it shouldnt be complaining at all
> Passenger can be also a child or infants.